### PR TITLE
Fix Esirkepov deposition in 1D

### PR DESCRIPTION
**Pull request by RemiLehe**
There were a few typos in the 1D code for Esirkepov.

This correction naturally leads to small changes in the benchmark files.

### CODE REVIEW

File: Regression/Checksum/benchmarks_json/LaserAcceleration_1d.json [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-    "particle_orig_z": 0.022432812500000038,
-    "particle_position_x": 0.02243266637270741,
-    "particle_regionofinterest": 40.0,
-    "particle_weight": 5.20625e+18
+    "particle_momentum_y": 0.0,
+    "particle_momentum_z": 0.0,
+    "particle_orig_z": 0.0,
+    "particle_position_x": 0.0,
+    "particle_regionofinterest": 0.0,
+    "particle_weight": 0.0
```
Comment (EZoni) [inline Regression/Checksum/benchmarks_json/LaserAcceleration_1d.json]: Note that these zeros might be related to #2819.

**Review by EZoni — APPROVED** [review]
Thank you for this fix, @RemiLehe! Looks good to me.

**Comment by EZoni** [comment]
Note that the checksums of the CI test `LaserAcceleration_1d` seem to need another reset.

It might be that our automatic tool to reset checksums does not work for particle data in 1D
(maybe because of some missing fix on the `yt` side).

Update: correct, the problem was only local, though.

**Review by roelof-groenewald — APPROVED** [review]
Thanks for looking into and fixing this issue. 
Fixes #3513.

**Pull request merged**